.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
done

Reply:
Even though you've added the file to the .gitignore it will probably still end up being pushed in this commit.

I think you'll have to `git rm -rf ` the `.idea` directory and any other files that shouldn't be added and then commit that. That will count as the line never being added. You may also need to locally remove all cached versions `git rm -rf . --cached`. Make sure you don't have any un-pushed pending changes, just in case the commands do something you weren't expecting.

Then you'll have 2 options. You can either squash all the commits here into one and force push it, or you can squash them when merging. If you don't do this we're going to have some commits adding changes that are later reverted, and it'll get messy